build-logic/jvm/src/main/kotlin/gradlebuild.launchable-jar.gradle.kts

```diff
@@ -46,6 +46,9 @@ tasks.jar.configure {
 val startScripts = tasks.register<GradleStartScriptGenerator>("startScripts") {
     startScriptsDir.set(layout.buildDirectory.dir("startScripts"))
     launcherJar.from(tasks.jar)
+    // The trick below is to use the templates from the current code instead of the wrapper. It does not cover the case where the generation logic is updated though.
+    unixScriptTemplate.from(layout.projectDirectory.file("../plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt"))
```

Comment:
`../plugins` seems a bit weird. Shall we use `project(":plugins").layout.projectDirectory.file(...)`?

Reply:
I wonder, as it means reaching out into another project though.

Reply:
If `layout` provided `rootDirectory` we could more naturally go from there.
+1 to keep it as is and not reach across projects. We're reaching across builds anyway :roll_eyes: